test/test_manifest.json

```diff
@@ -3137,6 +3137,13 @@
        "type": "eq",
        "annotations": true
     },
+    {  "id": "annotation-text-widget",
```

Comment:
I'd suggest adding `-forms` to the end of the test name, to make it obvious what's beeing tested.